colossalai/inference/config.py

```diff
@@ -43,8 +45,24 @@ class InferenceConfig:
     max_seq_len: Optional[int] = None
     quant_mode: Optional[str] = None
     revision: Optional[str] = None
-    # TODO: beam search is not support for now
     beam_width: int = 1
+    # TODO: beam search is not support for now
+    ratio: Optional[float] = 1.2
+    # the ratio of prefill sequences to decoding sequences, we do prefill step once the actual value exceeds ratio
+
+    def __init_batch_size__(self):
```

Comment:
This function is not called?

Reply:
`__xxx__` is the format of Python's built-in magic methods, please avoiding naming your function like this. Using `_init_batch_size` will be enough.

Reply:
done